tests/test_cape_result.py

```diff
@@ -576,9 +576,9 @@ def test_determine_dns_servers(network, inetsim_dns_servers, expected_result):
             # Domain is not safelisted
             ("blah.com", "1.1.1.1", [], {}, False),
             # Domain is safelisted
-            ("blah.ca", "1.1.1.1", [], {}, True),
+            ("blah.ca", "1.1.1.1", [], {}, False),
```

Comment:
Weird that "domain is safelisted" would change simply by extracting iocs as static... can you please investigate?